kolibri/content/management/commands/exportcontent.py

```diff
@@ -47,17 +47,12 @@ def handle_async(self, *args, **options):
 
                 with copy:
 
-                    with self.start_progress(total=copy.total_size) as file_cp_progress_update:
-
-                        for chunk in copy:
-                            if self.is_cancelled():
-                                copy.cancel()
-                                break
-                            length = len(chunk)
-                            overall_progress_update(length)
-                            file_cp_progress_update(length)
-                        else:
-                            exported_files.append(dest)
+                    size = copy.fast_file_copy()
```

Comment:
How do we handle file downloads now? I remember that file downloads use the loop above.

Reply:
Oh sorry, this is content exporting.

Reply:
Yeah, file download is the same as before.